vxdev/run-vxadmin.desktop

```diff
@@ -2,6 +2,6 @@
 Version=1.0
 Name=VxAdmin
 Icon=runprogram.png
-Exec=gnome-terminal -- /bin/sh -c 'env DEBUG=* env PATH=$PATH:/sbin env ADMIN_WORKSPACE=/vx/data/admin-service /vx/code/vxsuite-complete-system/run.sh election-manager'
+Exec=gnome-terminal -- /bin/sh -c 'env PATH=$PATH:/sbin env ADMIN_WORKSPACE=/vx/data/admin-service /vx/code/vxsuite-complete-system/run.sh election-manager'
```

Comment:
sidenote @amcmanus I had so much difficulty figuring out how to get these desktop entry files to launch our run script properly, I could not figure out a way where it would set these variables without having to put them here and also append the sbin to the path (even though it is in the users default path, it would disappear when running through this desktop file). I tried many different ways of calling this with gnome-terminal without that, etc. and this was the ultimate combination of things that seemed to work but I don't understand it and I'm sure there is a more straightforward way to do this.